lib/smart_answer_flows/state-pension-topup/questions/how_much_extra_per_week.govspeak.erb

```diff
@@ -6,6 +6,10 @@
   Enter a number between 1 and 25.
 <% end %>
 
-<% content_for :error_message do %>
-  Enter an amount in whole pounds between 1 and 25.
+<% content_for :error_not_whole_number do %>
+  Enter an amount which is a whole number.
```

Comment:
I thought that my wording might make it clearer that the user should not enter a "£" sign, but I'm happy to change it back if you think that's better.